openml-lightgbm/lightgbm-provider/src/test/java/com/feedzai/openml/provider/lightgbm/LightGBMBinaryClassificationModelTrainerTest.java

```diff
@@ -406,4 +552,17 @@ private static LightGBMBinaryClassificationModel fit(
             Files.delete(tmpModelFilePath);
         }
     }
+
+    /**
+     * Method to create a {@link ArrayList} with two {@link LinkedList}
+     *
+     * @return A {@link ArrayList} of two {@link LinkedList}.
+     * @since 1.2.2
+     */
+    private static ArrayList<List<Double>> getListOfTwoLists() {
```

Comment:
change to `private static List<List<Double>> getListOfTwoLists()`

Reply:
this is left intentionally as arraylist, so we can access it as arraylist.